### ISSUE TITLE
rebootstrap.pl fails with type check error

### ISSUE THREAD

**Issue by brianwisti** [tadzik/panda#250]
Came across this while trying to update rakudo & panda with rakudobrew.

`````
$ perl6 rebootstrap.pl
Type check failed in assignment to $prefix; expected Str but got CompUnit::Repository::Installation
  in sub MAIN at rebootstrap.pl:13
  in block <unit> at rebootstrap.pl:8
$ perl6 --version
This is rakudo version 2015.11-307-g5a9e0b1 built on MoarVM version 2015.11-21-g469ba64 implementing Perl v6.b.
`````

Pull request ~~pending~~ made. I wanted an issue number for the change.

**Comment by FROGGS** [comment]
This got fixed with #249

**Comment by brianwisti** [comment]
My mistake. Thanks!
